releasenotes/notes/kolla-kayobe-version-check-f44d43c9c34d1b89.yaml

```
@@ -0,0 +1,12 @@
+---
+features:
+  - |
+    Ansible playbooks to check the installed Kayobe/Kolla-Ansible versions
+    with the expected versions in Kayobe configuration. These checks
+    will run on Kayobe bootstrap, host and service operations.
```

nit: wording could be improved
```suggestion
    Added playbooks to check the installed Kayobe/Kolla-Ansible versions
    against the expected versions in Kayobe configuration. These checks
    will run on Kayobe bootstrap, host and service operations.
```